File: web/src/lib/billing-config.ts
/** Must match backend BILLING_ENABLED — payments are off until Stripe/Razorpay is integrated. */
export const billingConfig = {
  enabled: process.env.NEXT_PUBLIC_BILLING_ENABLED === 'true',
  freeTrialDays: Number(process.env.NEXT_PUBLIC_FREE_TRIAL_DAYS ?? '30') || 30,
  get trialMessage() {
    return this.enabled
      ? 'Simple plans when you are ready to upgrade'
      : `Everything free for ${this.freeTrialDays} days no payment needed`;
  },
  get launchBadge() {
    return this.enabled ? null : 'Free during early access';
  },
};

File: mobile/lib/core/constants/billing_config.dart
/// Mirrors web billing-config — payments off until billing is integrated.
class BillingConfig {
  static const bool enabled = false;
  static const int freeTrialDays = 30;

  static String get trialMessage => enabled
      ? 'Simple plans when you are ready to upgrade'
      : 'Everything free for $freeTrialDays days — no payment needed';

  static String? get launchBadge => enabled ? null : 'Free during early access';
}
